src/crate/client/sqlalchemy/dialect.py

```diff
@@ -194,8 +195,16 @@ def connect(self, host=None, port=None, *args, **kwargs):
             server = '{0}:{1}'.format(host, port or '4200')
         if 'servers' in kwargs:
             server = kwargs.pop('servers')
-        if server:
-            return self.dbapi.connect(servers=server, **kwargs)
+        servers = to_list(server)
+        if servers:
+            # Evaluate "ssl" connection URI query parameter.
+            # TODO: Evaluate more parameters like `ssl_ca`, `ssl_key`, `ssl_cert`,
+            #       `ssl_capath` and `ssl_cipher`.
```

Comment:
```suggestion
```

TODOs tend to stick around for way longer then they should. I think this should be tracked in an issue or just remove it. If it is important enough it will come up again.

Reply:
So true. 9984cc4826 removes the inline comment. Thanks.